plugins/reach/admin/forms/CatalogItemConfigure.php

```diff
@@ -126,6 +126,23 @@ public function init()
 			));
 		}
 		
+		if ($this->catalogItemType == Kaltura_Client_Reach_Enum_VendorServiceFeature::EXTENDED_AUDIO_DESCRIPTION)
```

Comment:
if both dubbing, audio, and audio description have the same elements the why not having in array condition and write them once instead of duplicating